### ISSUE TITLE
[Bug] RangeSlider in Scrollview (Android)

### ISSUE THREAD

**Issue by mikro1st** [xamarin/XamarinCommunityToolkit#502]
### Description
When implementing a RangeSlider inside a ScrollView, moving the upper and lower slider stops working after a little movement.

### Steps to Reproduce

1. Build A Contentpage like this:
`<?xml version="1.0" encoding="UTF-8"?>
<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             x:Class="MyApp.Test"
             xmlns:toolkit="clr-namespace:Xamarin.CommunityToolkit.UI.Views;assembly=Xamarin.CommunityToolkit">
    <ContentPage.Content>
        <ScrollView>
            <StackLayout VerticalOptions="Center" Margin="20">
                <toolkit:RangeSlider />
            </StackLayout>
        </ScrollView>
    </ContentPage.Content>
</ContentPage>`

2. Build the App with the TestView as MainPage
3. Try to move the Sliders

### Expected Behavior
RangeSliders should work properly inside ScrollViews

### Actual Behavior
RangeSliders thumbs stop moving after a bit of movement inside ScrollViews

### Basic Information

- Version with issue:
- Last known good version: -
- IDE: Visual Studio for Mac 8.7.8 build 4
- Platform Target Frameworks:
  - Android: 8.0  
- Nuget Packages: XamarinCommunityToolkit pre4 1.0.0
- Affected Devices: Huawei 10 lite

### Workaround
Did not found any yet, but will keep you informed if i do.

**Comment by AndreiMisiukevich** [comment]
Hi @mikro1st
does it work well on iOS?

**Comment by mikro1st** [comment]
Yes on an iPhone 8 iOS 14.0.1 it works as predicted

**Comment by mikro1st** [comment]
What i figured out this has to to with slight vertical movements while swiping the upper or lower thumb.
while you are only moving sideways, every thing is fine. As soon as there is a minimal vertical movement recognized it will break.

**Comment by AndreiMisiukevich** [comment]
@mikro1st well, what will happen if you will set IsEnabled = false to the ScrollView on DragStarted and return to True on DragEnded?

**Comment by mikro1st** [comment]
@AndreiMisiukevich maybe I'm to bad as an Programmer...
I tried DragAndDrop as well as Pan GestureRecognizers like this:

XAML:
`<ScrollView x:Name="scrollview">
        <StackLayout VerticalOptions="Center" Margin="20">
            <toolkit:RangeSlider>
                <toolkit:RangeSlider.GestureRecognizers>
                   <PanGestureRecognizer PanUpdated="PanGestureRecognizer_PanUpdated"/>
<DragGestureRecognizer DragStarting="DragGestureRecognizer_DragStarting" DropCompleted="DragGestureRecognizer_DropCompleted"/>
                </toolkit:RangeSlider.GestureRecognizers>
            </toolkit:RangeSlider>
        </StackLayout>
    </ScrollView>`

CodeBehind:
`void PanGestureRecognizer_PanUpdated(System.Object sender, Xamarin.Forms.PanUpdatedEventArgs e)
 {
 scrollview.IsEnabled = e.StatusType != GestureStatus.Running;
 }`

 `void DragGestureRecognizer_DragStarting(System.Object sender, Xamarin.Forms.DragStartingEventArgs e)
 {
 scrollview.IsEnabled = false;
 }`

 `void DragGestureRecognizer_DropCompleted(System.Object sender, Xamarin.Forms.DropCompletedEventArgs )        {
 scrollview.IsEnabled = true;
 }`

What I did recognize: the Gestures will only have an effect, when i pan or drag and drop the slider bar. But it will not be fired when i drag the thumbs. 
I was not abled to implement an GestureRecognizer for only the LowerThumbView. This resulted in an error inside the XAML.

**Comment by AndreiMisiukevich** [comment]
I meant DragStarted/DragEnded events of RangeSlider that were added here https://github.com/xamarin/XamarinCommunityToolkit/pull/497

**Comment by mikro1st** [comment]
Understood.

Upgraded ToolKit to 1.0.0-preview1082
Inserted DragStarted and DragCompleted Eventhandler, setting the IsEnabled of the ScrollView
=> Error is still there.

What I recognized:
The DragCompleted Event will not be Fired, when I run into this Bug.
I have to do tiny movements on the Thumb to fire the DragCompleted Event.
When the DragCompleted Event wasn't fired, the DragStarted Event will not fire anymore (which seems logical)

I then set the IsEnabled Property to false without changing it anymore
=> Error is still there.
The Rangeslider does not care about whether the Scroll view is Active or not. but it doesn't like to have a Parent which can move the page.
I have the feeling that also my CarouselPage will make Errors later on. But lets focus on the ScrollView here. :)
